distro/connect-converter/pom.xml

```diff
@@ -0,0 +1,125 @@
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.apicurio</groupId>
+        <artifactId>apicurio-registry-distro</artifactId>
+        <version>1.1.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>apicurio-registry-distro-connect-converter</artifactId>
+    <name>apicurio-registry-distro-connect-converter</name>
+
+    <properties>
+        <assembly.descriptor>converter-distribution</assembly.descriptor>
+        <version.jersey>2.29.1</version.jersey>
+        <version.geronimo.config>1.2.2</version.geronimo.config>
+        <distro.version.kafka>2.4.0</distro.version.kafka>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.apache.kafka</groupId>
+                <artifactId>connect-json</artifactId>
+                <version>${distro.version.kafka}</version>
```

Comment:
Shouldn't this be inlined with what we have in top pom.xml ?

Reply:
@alesj No, this is intentional as this depends on to what Kafka Connect version you want to deploy. Moreover you have desynced Kakfa and Kafka Connect. Kafka clients are brought from Quarkus in version 2.3.x and you are using Connect in version 2.2.x
It is not possible to align both on the same version -even if it is desirable as you are using Connect 2.2.x specific API and upgrading breaks the build.

Reply:
@jpechane OK, I see ... 
Yeah, we depend on Quarkus ... so we can move the Connect to the same 2.3.x ?

Reply:
@alesj You can but the code needs to be updated then - the file `io/apicurio/registry/connector/ConnectorApplication.java`.

Just a sidenote, maybe you can look at `io.debezium.embedded.EmbeddedEngine` there might be a place for cooperation